packages/report-flaky-tests/src/markdown.ts

```diff
@@ -96,6 +96,7 @@ function formatTestErrorMessage( flakyTestResult: FlakyTestResult ) {
 		default: {
 			return stripAnsi(
 				formatResultsErrors(
+					// @ts-ignore
```

Comment:
Let's remove this from the pull request because it's an unrelated change.

If it's a necessary change, you can submit a separate PR for it.

Reply:
I don't know why this happens, and if it's only me. But without it, I have to push with `-no_verify`. I'll remove that, and see how it goes, as it wasn't introduced by me.

Reply:
I don't know why this happens, and if it's only me. But without it, I have to push with `-no-verify`. I'll remove that, and see how it goes, as it wasn't introduced by me.